python/graphstorm/config/argument.py

```diff
@@ -227,6 +227,14 @@ def save_perf_results_path(self):
             return self._save_perf_results_path
         return None
 
+    @property
```

Comment:
You also need to update docs.source.configuration

Reply:
what do you mean?